### DISCUSSION TITLE
OpenHayStack vs FindMy on real Mac, what is the best?

### DISCUSSION THREAD

**Discussion by paolostefanucci** [Chapoly1305/FindMy#15] [Q&A] [answered]
If I have a real Mac Mini M2 computer (Arm Architetture), so no virtualization, what solution is best and durable in the time, OpenHayStack or FindMy project?
I am not interesting on the GUI, but rather on the underlying system.
Is it better the system with Apple Mail Plugin implemented by OpenHayStack, or
is better the underlying system for communication to Apple FindMy Network implemented by this FindMy project?
What system are more vulnerable to be banned and closed by Apple in future?

**Comment by Chapoly1305** [comment, marked as answer]
Sorry for the late response, Github notification might failed to inform me about this question. 

**OHS original design used Apple Mail Plugin which is no longer supported since MacOS 14.** I am not sure what's the stock OS of your Mac Mini, if it's 14 or above it won't work. I heard OHS has adapted new strategy to workaround this issue. 

I actually don't have answer about which one is less risky getting banned. But I do know which cost less damage. Considering the **Mail Plugin method actually using a real device identity and Apple can know the real identity sending requests**. This repo however using made up identity so as long as you are using an Apple ID unable to trace back, you could archives a degree of anonymity. Most importantly, you don't need to concern a real Apple product getting banned.
